src/shared/featureFlags.ts

```diff
@@ -28,6 +28,12 @@ export const flags: Record<Feature, IFeatureFlag> = {
     experimentName: 'Customer Single Sign-On',
     persist: true,
   },
+  NextGenClusters: {
+    enabled: false,
+    init: () => window.featureFlags.FEATURE_MAPI_CLUSTERS,
+    experimentName: 'Next-Gen Clusters',
```

Comment:
I'd like to make the experiment label more descriptive.

```suggestion
    experimentName: 'Use Management API for the list of clusters',
```

Reply:
Uhh, that's not as exciting 😢